proxy/http/HttpTransact.cc

```diff
@@ -3022,6 +3022,12 @@ HttpTransact::HandleCacheOpenReadMiss(State *s)
   HTTPHdr *h = &s->hdr_info.client_request;
 
   if (!h->is_cache_control_set(HTTP_VALUE_ONLY_IF_CACHED)) {
+    // Initialize the server_info structure if we haven't been through DNS
+    // Otherwise, the http_version will not be initialized
+    if (!s->current.server || !s->current.server->dst_addr.isValid()) {
+      s->server_info.http_version = HTTPVersion(0, 0);
```

Comment:
Why are we initializing it to (0, 0)?  Where in the code is it defaulting to HTTP/1.0?